Project/Source/Components/ComponentParticleSystem.cpp

```diff
@@ -891,8 +1021,20 @@ void ComponentParticleSystem::UpdateGravityDirection(Particle* currentParticle)
 	currentParticle->direction = float3(x, y, z);
 }
 
+void ComponentParticleSystem::UpdateTrail(Particle* currentParticle) {
+	currentParticle->trail->Update(currentParticle->position);
+}
+
+void ComponentParticleSystem::UpdateWitdhTrail() {
```

Comment:
Not used. Is it for the change of width from ImGui? Probably it's useful also from Gameplay, if you want to add it